### PR TITLE
Fix navbar hamburger icon in Community page

### PR DESCRIPTION
**Pull request by anurag4499**
Navbar hamburger icon overlaps with white space in "Community" page is solved by mine.
plz check it.

### CODE REVIEW

**Comment by anurag4499** [comment]
[annajung](https://github.com/annajung) @annajung 
plz approve

**Comment by tengqm** [comment]
We are fixing an existing issue with this PR, right?

File: static/css/community.css [hunk 1]
```diff
@@ -394,8 +394,40 @@ body.cid-community #cncf-code-of-conduct h2:after {
...
+  body.cid-community .community-section h2:before,
+body.cid-community .community-section h2:after {
+  background-color: #aaaaaa;
+  content: "";
+  display: inline-block;
+  height: 1px;
+  position: relative;
+  vertical-align: middle;
+  width: 7%;
+}
```
Comment (tengqm) [inline static/css/community.css:423]: Some further indentation would help improve readability, agree?

File: static/css/community.css [hunk 2]
```diff
@@ -394,8 +394,40 @@ body.cid-community #cncf-code-of-conduct h2:after {
...
+}
+
+
+
+
+
+
```
Comment (tengqm) [inline static/css/community.css:430]: This empty lines can be dropped.

File: static/css/community.css [hunk 3]
```diff
@@ -394,8 +394,40 @@ body.cid-community #cncf-code-of-conduct h2:after {
...
+  body.cid-community .community-section h2:before,
+body.cid-community .community-section h2:after {
+  background-color: #aaaaaa;
+  content: "";
+  display: inline-block;
+  height: 1px;
+  position: relative;
+  vertical-align: middle;
+  width: 18%;
+}
```
Comment (tengqm) [inline static/css/community.css:407]: Maybe some more indentations are desirable?

File: static/css/community.css (hunk 1 above)
Comment (niranjandarshann) [reply static/css/community.css:423]: At lines 408 and 409 there are empty lines , these lines can be dropped.

**Comment by niranjandarshann** [comment]
@anurag4499  
Could you please squash your commits: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by Gauravpadam** [comment]
Issue pertaining to this (https://github.com/kubernetes/website/issues/42854) has been fixed by another PR

/close

**Pull request closed without merging**